homeassistant/components/airtouch4/__init__.py

```diff
@@ -0,0 +1,109 @@
+"""The AirTouch4 integration."""
+import logging
+
+from airtouch4pyapi import AirTouch
+import voluptuous as vol
+
+from homeassistant.components.climate import SCAN_INTERVAL
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+CONFIG_SCHEMA = vol.Schema(
+    {DOMAIN: vol.Schema({vol.Required(CONF_HOST): str})}, extra=vol.ALLOW_EXTRA
+)
```

Comment:
AFAIK, we are not permitted to use config schemas any longer. There is an ADR about this.